deploy/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.24
```

Comment:
I don't think that "going back to the behavior from 1.24" makes sense for a feature that changed on purpose in k8s. Can we fix it differently? This seems like adding tech debt.

Reply:
Read more about it, and it makes sense to use v1.24 here. Ignore my previous message.

Reply:
Can we use `latest` instead of v1.24?

Reply:
Reading more into this, it seems that `enforce-version` is optional. It's not clear to me why we need it in the first place?

Reply:
I think we can remove that part not problem